src/examples/java/com/amazonaws/crypto/examples/SetCommitmentPolicyExample.java

```diff
@@ -3,17 +3,17 @@
 
 package com.amazonaws.crypto.examples;
```

Comment:
All of the examples in this directory are now using deprecated methods. I would prefer to move these to another location (maybe `deprecated` or `v2`?).